src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,34 @@
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const NavBar:React.FC = () => {
+  return (
+    <nav
+      className="navbar is-light is-fixed-top is-mobile has-shadow"
+      data-cy="Nav"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            to="/"
+            className={({ isActive }) => classNames(
+              'navbar-item',
+              { 'is-active': isActive },
+            )}
+          >
+            Home
+          </NavLink>
+          <NavLink
+            to="/tabs"
+            className={({ isActive }) => classNames(
+              'navbar-item',
+              { 'is-active': isActive },
+            )}
```

Comment:
it would be better to create components and pass path and title to them, like this:
        <NavItem to="/" title="Home" />
        <NavItem to="/tabs" title="Tabs" />
But it's up to you. Your version is ok too.

Reply:
Yeap, you could create a component that takes the path and a link title.